cartridges/int_bolt_sfra/cartridge/client/default/js/boltModalConfigure.js

```diff
@@ -4,4 +4,5 @@ var bolt = require('./bolt');
 
 $(document).ready(function () {
     bolt.initBoltButton();
+    bolt.addApplePayHandlerIfNeeded();
```

Comment:
Can we add some comments here? like `ignore this if you are not using ApplePay.` so that merchants can aware of it.